express-static/server.ts

```diff
@@ -0,0 +1,12 @@
+import express from 'express';
+
+const app = express();
+
+app.get('/', (req, res) => {
+  res.send('Hello World');
+});
+
+app.use(express.static('public'));
```

Comment:
Usually the static middleware would be mounted before the routes are declared. Order matters in express middleware, though in this case it likely won't make a difference.

Reply:
understood, from the discussion today, that makes total sense as static doesn't call next middleware. if it fulfills a promise, i will keep that in mind going forward.

Thank you!